stats/stats_test.go

```diff
@@ -59,8 +59,10 @@ func init() {
 	grpc.EnableTracing = false
 }
 
-type connCtxKey struct{}
-type rpcCtxKey struct{}
+type (
```

Comment:
We don't declare types like that anywhere in our codebase so let's revert this as its not relevant to the change?

Reply:
My bad. Remnant of an gofmt mistake.